Makefile

```diff
@@ -9,9 +9,10 @@ local-release:
 test: build/turnkey
 	go test ./...
 
+.PHONY: build/turnkey
 build/turnkey: main.go internal/
 	go build -o build/turnkey main.go
```

Comment:
If this becomes a phony target, let's keep the shorter `build` name and remove the file/folder deps?
```makefile
.PHONY: build
build:
	go build -o build/turnkey main.go
```

Fewer characters to type!